package.json

```diff
@@ -25,5 +25,8 @@
   "bugs": {
     "url": "https://github.com/Grade-Notifier/GN-Core/issues"
   },
-  "homepage": "https://github.com/Grade-Notifier/GN-Core#readme"
+  "homepage": "https://github.com/Grade-Notifier/GN-Core#readme",
```

Comment:
Can we get rid of this?

Reply:
I guess? I didnt make this

Reply:
jsrsasign this is not you? @ericshermancs